playbooks/openshift-logging/private/config.yml

```diff
@@ -16,7 +16,7 @@
 # Normally we only collect this information for our master group entries
 # we want to also collect this for nodes so we can match group entries to nodes
 - name: Get common IP facts when necessary
-  hosts: all
+  hosts: oo_nodes_to_config:!oo_masters
```

Comment:
I don't know the scope of the data you need, but etcd hosts are not necessarily masters, they could be rpm installed.

Reply:
We are looking to collect `openshift.common.ip` for nodes so that we can match the private_ip of a node's address back to the group entry.
 
Isn't every thing a node, it just may also be an etcd host, nfs host, or master?

Reply:
etcds are not necessarily nodes.  Of course, if you're only interested in labeling some nodes or doing something to only nodes, obviously we don't need to account for etcd specifically.

Reply:
Correct, this is only something that we need to worry about with nodes. We need to know which nodes we'll be creating sysctl files for.